chemprop/uncertainty/calibrator.py

```diff
@@ -79,57 +80,220 @@ def apply(self, preds: Tensor, uncs: Tensor) -> tuple[Tensor, Tensor]:
 
 @UncertaintyCalibratorRegistry.register("conformal-multilabel")
 class ConformalMultilabelCalibrator(UncertaintyCalibrator):
+    r"""Creates conformal in-set and conformal out-set such that for :math:`1-\alpha` proportion of datapoints,
+    the set of labels is bounded by the in-set and out-set. That is said, the conformal in-set is contained
+    in the set of actual labels and the set of actual labels is contained in the conformal out-set. [1]_
```

Comment:
Somehow, the link cannot be shown correctly.
![image](https://github.com/user-attachments/assets/3362dcf3-2b38-4665-9bf2-6bf31b007636)

Reply:
Issue related to #982